catkin_ws/src/propulsion/embedded/thrusters/thrusters.ino

```diff
@@ -77,7 +83,7 @@ void initThrusters(){
 	thrusters[HVE_ST_P].attach(HVE_ST_P_PIN);
 
 	//set initial thruster effort (OFF)
-	thrustersOff();
+	//thrustersOff();
```

Comment:
what was the issue with setting the thrusters to initially be off?

Reply:
That's the initialization step. Setting it to zero here and then creating a publisher to the topic makes it re-initialize and lose sync.